### PR TITLE
[MODULAR]IPCs now act like Baycode

### PR DESCRIPTION
**Pull request by Vizoplez**
## About The Pull Request
Begins to layout the framework to make IPCs more bay-like and less copy-pasted carbons with a funny hunger mechanic.

- [ ] Make IPCs properly take burn damage from space
- [x] Add better mechanics and moderation techniques for this

## Why It's Good For The Game
Balancing, more unique features and items for IPCs. General WIP.

## Changelog
:cl:
tweak: IPCs can no longer transfer heat in ultra low pressure environments. Be careful out there, stalker.
tweak: All /robotic/ subtypes now take burn damage when above a certain temp
tweak: All robotic subtypes are now space immune, but take increasing heat in low pressures (mitigated by cold atmospheres such as icebox so you dont explode, thank me later)
icon: Adds cooler sprites from bay, thanks guys! All credit where its due.
/:cl:

### CODE REVIEW

**Comment by Vizoplez** [comment]
if it compiles it works

File: modular_skyrat/modules/modular_IPC/items/cooling_units.dm [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        update_power()
+    else
+        on = TRUE //im so fucking lazy
+        update_power()
+
+/obj/item/coolingpack/examine(mob/user)
+    . = ..()
+    . += "<span class='warning'>Charge Left: [cell.charge]/[cell.maxcharge]</span>"
+
+/obj/item/coolingpack/equipped(mob/user, slot)
```
Comment (MLGTASTICa) [inline modular_skyrat/modules/modular_IPC/items/cooling_units.dm]: This is called when its equipped , not every tick , so IPC's would need to constantly re-equip it to get cooling..

Comment (Vizoplez) [reply modular_skyrat/modules/modular_IPC/items/cooling_units.dm]: I appear to have retardation

**Comment by Vizoplez** [comment]
Drafting it again because /some/ code works

File: modular_skyrat/modules/modular_IPC/items/cooling_units.dm [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/modular_IPC/items/cooling_units.dm]: why

Comment (Vizoplez) [reply modular_skyrat/modules/modular_IPC/items/cooling_units.dm]: I have braindeath

**Comment by Vizoplez** [comment]
Closing due to technical issues for now, may be continued in the future w/e.

**Pull request closed without merging**